rules/properties-order/tests/flat.js

```diff
@@ -199,9 +237,19 @@ testRule({
 		{
 			code: 'a { top: 0; height: 1px; color: pink; }',
 		},
+		{
```

Comment:
Could remove this one and every other test below, that was added in this file .

#78 is about very minor edge case. We don't need so many tests. Code will be already covered by other tests.

But I appreciate your attention to details :)